numba/np/is_comparison_tests.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
```

Comment:
Whilst it's good to have added tests, they should be in the `numba.tests` module, and should fit in with the existing unit test framework.